### ISSUE TITLE
Implement multi-output reduction strategy for ReducedRegressionForecaster

### ISSUE THREAD

**Issue by mloning** [alan-turing-institute/sktime#403]
We currently support the following strategies for reduction from forecasting to regression (see this [file](https://github.com/alan-turing-institute/sktime/blob/master/sktime/forecasting/compose/_reduce.py)): 

- [x] direct
- [x] recursive
- [ ] dirrec (see #226)
- [ ] multi-output (see #440)

For an overview of forecasting strategies for multi-step forecasting horizons see this [paper](https://d1wqtxts1xzle7.cloudfront.net/41231210/Machine_Learning_Strategies_for_Time_Ser20160114-28513-1pqgoe9.pdf20160115-19908-phg1xc.pdf?1452888125=&response-content-disposition=inline%3B+filename%3DMachine_Learning_Strategies_for_Time_Ser.pdf&Expires=1602491710&Signature=V6cHqybz3g9idBMTkbC4kwDMppSA4FnO9cd~R0VY8JVmVUKrK28MGRJlKhMfODdsSxcWsT5wvu2y3elqoES3ILHn1vlka20fO9z~dYmKB4oPsV04fG3lRhDe1Sx5QKgSmU4PMpWxa3bAwE8DKC4dlDcuLY7wequBDkTlItBFeYqOrJUhc91tMJxMpJ7gVtDzmAFyP58Fw5iXQN19fhCe2RoMtQ2uLRJrWvmlEdRvOhrffgnOR1g9N71Nt01I3X5~DON8gmKEmaGKYWv1wYg7JrSmLXnyZ2gVBUgdhERUxZ7VKSOMp-z0KFsuPV1bL7MbIZrHs7eHMnXK9v4vqT47ig__&Key-Pair-Id=APKAJLOHF5GGSLRBV4ZA).  This issue is about the multi-output strategy. Some models can directly predict multiple outputs, see e.g. LinearRegression:
```python
import numpy as np
from sklearn.linear_model import LinearRegression

y = np.random.normal(size=(10, 3))  # target has multiple columns
X = np.random.normal(size=(10, 5))

estimator = LinearRegression()
estimator.fit(X, y)
y_pred = estimator.predict(X)  # we predict multiple columns
print(y_pred.shape)
>>> (10, 3)
```
 
We need to enable the multi-output prediction strategy by adding a "multioutput" option to the "strategy" kwarg in the [`ReducedRegressionForecaster`](https://github.com/alan-turing-institute/sktime/blob/master/sktime/forecasting/compose/_reduce.py).

We could check the estimator by checking if it inherits from scikit-learn's `MultiOutputMixin`.

**Comment by NikeNano** [comment]
I would be happy to contribute this, can I pick it up?

**Comment by NikeNano** [comment]
take

**Comment by mloning** [comment]
Hi @NikeNano yes thanks! No-one is currently working on this particular issue, but note that there is ongoing development for the dirrect and extending the existing strategies to multivariate versions. 

But this issues seems to be sufficiently separated to start working on it. Let me know if you need any help or have any questions!

**Comment by NikeNano** [comment]
Thank you  @mloning, I will start to work on it tomorrow. Will get back to you if I have any questions.

**Comment by NikeNano** [comment]
@mloning is it possible to get access to the article, the link is not working for me  due to : ``` AccessDeniedAccess denied```

**Comment by mloning** [comment]
@NikeNano I've fixed the link!

**Comment by NikeNano** [comment]
I will not have time to work on this in the near future, so if any one like to pick it up feel free to do so.

**Comment by Lovkush-A** [comment]
@mloning Can you assign me to this issue?

**Comment by mloning** [comment]
Closed by #659
